lib/kube/proxy/testing/kube_server/kube_mock.go

```diff
@@ -335,12 +348,15 @@ func createRemoteCommandProxy(req remoteCommandRequest) (*remoteCommandProxy, er
 		err   error
 	)
 	if wsstream.IsWebSocketRequest(req.httpRequest) {
-		return nil, fmt.Errorf("only SPDY streams upgrades are supported")
-	}
-
-	proxy, err = createSPDYStreams(req)
-	if err != nil {
-		return nil, trace.Wrap(err)
+		proxy, err = createWebSocketStreams(req)
```

Comment:
kube service should never forward the request to kube api as websocket without properly ensuring it supports websocket v5